packages/app/src/specs/SpecsList.vue

```diff
@@ -413,6 +433,10 @@ const { refetchFailedCloudData } = useCloudSpecData(
   props.gql.currentProject?.specs as SpecsListFragment[] || [],
 )
 
+const hasRuns = ref(false)
+
+watch([list], () => hasRuns.value = false)
```

Comment:
```suggestion
watch(list, () => hasRuns.value = false)
```

Reply:
Also I don't fully understand the logic here - when list changes, we set `hasRuns` to `false` - why do we do that?